composer.json

```diff
@@ -1,26 +1,27 @@
-{
-    "name": "ivantage/ivantagejobqueue",
-    "description": "iVantage Job Queue",
-    "version": "1.0.0",
-    "keywords": [
-        "zend",
-        "job",
-        "queue"
-    ],
-    "homepage": "http://ivantagehealth.com/",
-    "require": {
-        "php": ">=5.3.3",
-        "zendframework/zendframework": "~2.4.0"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "4.4.*"
-    },
-    "autoload": {
-        "psr-0": {
-            "IvantageJobQueue": "src/"
-        },
-        "classmap": [
-            "./Module.php"
-        ]
-    }
-}
+{
+    "name": "ivantage/ivantagejobqueue",
+    "description": "iVantage Job Queue",
+    "version": "1.0.0",
+    "keywords": [
+        "zend",
+        "job",
+        "queue"
+    ],
+    "homepage": "http://ivantagehealth.com/",
+    "require": {
+        "php": ">=5.3.3",
+        "zendframework/zend-mvc": "^3.1",
+        "zendframework/zend-db": "^2.11"

```

Comment:
Do you know if this package is needed? I don't recall using anything from the Zend\Db namespace.